### ISSUE TITLE
Elixir 1.0.3 and Erlang OTP 17.5 not working

### ISSUE THREAD

**Issue by yeradis** [elixir-lang/elixir#3228]
Trying on Windows 8.1 x64

1) Install Erlang OTP 17.5
2 Install Elixir 1.0.3 using web setup

Then trying to get the version you can see this:
```
C:\elixir>elixir -v
{"init terminating in do_boot",{{badmatch,{error,{elixir,{bad_return,{{elixir,start,[normal,[]]},{'EXIT',{{badmatch,{error,enotsup}},[{elixir,start,2,[{file,"src/elixir.erl"},{line,34}]},{application_master,start_it_old,4,[{file,"application_master.erl"},{line,272}]}]}}}}}}},[{elixir,start_cli,0,[{file,"src/elixir.erl"},{line,75}]},{init,start_it,1,[{file,"init.erl"},{line,1057}]},{init,start_em,1,[{file,"init.erl"},{line,1037}]}]}}

Crash dump was written to: erl_crash.dump
init terminating in do_boot ()
```

Also:

```
C:\elixir>iex

=INFO REPORT==== 1-Apr-2015::22:58:56 ===
    application: elixir
    exited: {bad_return,
                {{elixir,start,[normal,[]]},
                 {'EXIT',
                     {{badmatch,{error,enotsup}},
                      [{elixir,start,2,[{file,"src/elixir.erl"},{line,34}]},
                       {application_master,start_it_old,4,
                           [{file,"application_master.erl"},{line,272}]}]}}}}
    type: temporary

=INFO REPORT==== 1-Apr-2015::22:58:56 ===
    application: syntax_tools
    exited: stopped
    type: temporary

=INFO REPORT==== 1-Apr-2015::22:58:56 ===
    application: compiler
    exited: stopped
    type: temporary

=INFO REPORT==== 1-Apr-2015::22:58:56 ===
    application: crypto
    exited: stopped
    type: temporary

=ERROR REPORT==== 1-Apr-2015::22:58:56 ===
Error in process <0.24.0> with exit value: {{badmatch,{error,{elixir,{bad_return,{{elixir,start,[normal,[]]},{'EXIT',{{badmatch,{error,enotsup}},[{elixir,start,2,[{file,"src/elixir.erl"},{line,34}]},{application_master,start_it_old,4,[{file,"application_master....
```

**Comment by josevalim** [comment]
Yes, it is fixed in v1.0 and master branches. v1.0.4 is going to be released by next week.

**Comment by jeznet** [comment]
Till 1.0.4 comes out you can try: https://groups.google.com/forum/#!topic/elixir-lang-talk/FyzHE6N16Nk
